artnet/plugin.yaml

```diff
@@ -61,14 +63,22 @@ parameters:
             en: 'Specifies the Art-Net Universe to use'
         valid_min: 0
         valid_max: 15
+    min_channels:
+        type: int
+        default: 1
+        description:
+            de: 'Gibt an, wie viele Kanäle mindestens gesendet werden müssen, hilfreich für Geräte, die einen vollen Datensatz über alle ihre Kanäle erhalten müssen'
+            en: 'Specifies how many DMX channels are to be sent at minimum'
+        valid_min: 1
+        valid_max: 512
```

Comment:
What the reason for a min-channel settings. As the code tells me, it's only used to fillup the data buffer which gets send to the DMX network. Before it was dynamically increased depending on value changes to the addresses: if the data buffer does not contain a value for the given address, the buffer will be increased up to the given address and the new value will be put into the data buffer.

We could also initialize the complete data buffer for all addresses when parsing items finished. So when the plugin identified 64 as the maximum address the plugin could initialize the data buffer with 64 values (initial value set to zero). When now sending the data to the DMX network it will always send the 64 values to it. No need to specify an explicit value for minimal channels to send.

Maybe I oversee something, but I can't imagine a benefit on having such an attribute when we also could have a default initialization of the data buffer with the maximum address used in configuration.

Reply:
This is to avoid issues as described here:
https://knx-user-forum.de/forum/supportforen/smarthome-py/996278-problem-mit-artnet-plugin-bzw-der-dmx-hardware

Imagine a setup as follows: 
- One 36-channel dimmer (to be obtained easily and quite cheap). 
- Only e.g. 32 channels are used and the 33rd till 36th channel are left unconnected. Economic solution as there is no cheaper way to control 8 rgbw lights, e.g. in a garden-setup.
- Only items are specified, no logic is run.

So if you send "all LEDs white" no light will light up as the dimmer requires all 36 channels to be defined. To avoid this you can use this config to ensure the transmission of at least n channels. Otherwise ignore it.